janitor/functions/utils.py

```diff
@@ -693,33 +695,67 @@ def _process_SD(df, arg):
     return columns, names, zip(func_names, func), dupes
 
 
-class SD:
+class col:
```

Comment:
@samukweku I'm finally waking up to how creative this implementation is! :smile: :heart: